### ISSUE TITLE
Headers in contigs from metaplasmidspades

### ISSUE THREAD

**Issue by eperezv** [ablab/spades#475]
Hello,
I have problems to interpret the header from a "contig.fasta" file from spades.py 3.14.0 with the "--meta --plasmid" options. Basically, I get something like this:

```
>NODE_1_length_XXXXX_cov_XXXXX_cutoff_0
>NODE_2_length_XXXXX_cov_XXXXX_cutoff_0
>NODE_3_length_XXXXX_cov_XXXXX_cutoff_0
>NODE_1_length_XXXXX_cov_XXXXX_cutoff_42
>NODE_2_length_XXXXX_cov_XXXXX_cutoff_42
and so on
```

I do not understand why I have several "NODE_1" headers. This behavior seems different from the standard options and only happens with the plasmid mode. In the manual, it is indicated that a suffix _component_X should be appended, but I only saw see "cutoff". What does this cutoff mean?

Thanks

**Comment by Dmitry-Antipov** [comment]
Hi.
Unfortunately all updates to manual regarding metaplasmidSPAdes are missing in 3.14.0 release - see  http://cab.spbu.ru/files/release3.14.0/manual.html#sec3.6 
However, now I see than even updated version may be not clear, we'll fix it in 3.14.1

About NODE_ and _cutoff_ in headers - metaplasmidSPAdes does several iterations of coverage-based removal of chromosomal contigs using different coverage cutoffs, so for each coverage cutoff (_cutoff_X) there are it's own NODE_1_ NODE_2, etc. 

More detailed explanation of the algorithm may be found here  https://genome.cshlp.org/content/29/6/961.full.pdf

**Comment by eperezv** [comment]
Hi,

Thanks a lot for your answer and clarification. I was confused because naming "similarly" different contigs is not as in the standard version of spades.

**Comment by asl** [comment]
Fixed in 3.14.1
